src/library/trackset/crate/cratetablemodel.cpp

```diff
@@ -74,20 +76,20 @@ bool CrateTableModel::addTrack(const QModelIndex& index, QString location) {
             TrackRef::fromFileInfo(fileInfo));
     if (!pTrack) {
         qDebug() << "CrateTableModel::addTrack:"
-                << "Failed to add track"
-                << location
-                << "to library";
+                 << "Failed to add track"
```

Comment:
These indentation changes are inconsistent with our .clang-form. Applies to multiple occurrences in this PR!

Reply:
Nope, I have applied clang-format to everything! 
I double-checked for this file, re-running clang-format changes nothing.

Reply:
Double-checked again and confirmed. Strange, I remember that this was indented with 8 spaces instead of aligning the `<<` operators on subsequent lines.